test/unit/schema.js

```diff
@@ -547,14 +626,17 @@ module.exports.tests.api_services_validation = (test, common) => {
 
   });
 
-  test('when api.services.placeholder is defined, url is required', (t) => {
+};
+
+module.exports.tests.pip_service_validation = (test, common) => {
+  test('when api.services.pip is defined, url is required', (t) => {
     var config = {
       api: {
         version: 'version value',
         indexName: 'index name value',
         host: 'host value',
         services: {
-          placeholder: {
+          pip: {
```

Comment:
I think it's interpreting the diff strangely.  Basically, there are placeholder and pip services schema tests that test the same functionality.